packages/docs/docs/dev/abis.mdx

```diff
@@ -7,16 +7,16 @@ category: For Developers
 
 ## Get ABI of contract ID
 
-To get the ABI of a contract ID use the `getAbi('contractId')` method on the `window.fuel` object.
+To get the ABI of a contract ID use the `getAbi` with `contractId`.
 
-<CodeImport file="../../examples/Abi.tsx" />
+<CodeImport file="../../examples/abis/Abi.tsx" commentBlock="getABI" />
 
 <Examples.Abi />
 
 ## Adding ABI
```

Comment:
Makes more sense to put "Adding ABI" first and "Get ABI of contract ID" after. I was clicking get ABI of contracts and it returns null and displays nothing (as expected) but developers new to blockchain wouldnt know that.